build/ci-build.yml

```diff
@@ -1,113 +1,56 @@
-name: $(date:yyyyMMdd)$(rev:.r)
+# Starter pipeline
 
-trigger:
```

Comment:
You'll want to use this trigger instead of yours, it won't run against PRs

Reply:
yes, took most of the standard template